snapcraft/tests/test_common.py

```diff
@@ -23,6 +23,17 @@
 
 class CommonTestCase(tests.TestCase):
 
+    def setUp(self):
+        super().setUp()
+        self.addCleanup(self._reset_default_paths)
+
+    def _reset_default_paths(self):
```

Comment:
Good catch! I have used that opportunity to replace the duplicate `    self.addCleanup(common.set_schemadir, common.get_schemadir())` with the tourdir cleanup. I added the librariedir cleanup that wasn't there too. Just pushed the new revision.